src/main/javadoc/overview.html

```diff
@@ -0,0 +1,362 @@
+KinesisVideoClient client = KinesisVideoJavaClientFactory.createKinesisVideoClient(...);
+
+// 2. Create a media source (sending video frames in a loop)
+//    A media source represents a Kinesis Video Stream
+MediaSource mediaSource = new YourMediaSource();
+
+// 3. Start streaming (start the putFrame loop)
+client.registerMediaSource(mediaSource);
+mediaSource.start();</pre>
+<p><strong>That's it!</strong> Your video is now streaming to AWS. See the demo apps for complete examples.</p>
+
+<h3>Demo Applications</h3>
+<ul>
+    <li><strong>DemoAppMain</strong> - Streams sample H.264 video files using
+        <strong>com.amazonaws.kinesisvideo.java.mediasource.file.ImageFileMediaSource</strong></li>
+</ul>
+
+<div>
+    <h3>Prerequisites</h3>
+    <ul>
+        <li><strong>Java:</strong> Version 11, 17, or 21</li>
+        <li><strong>Maven:</strong> For building the project</li>
+        <li><strong>AWS Credentials:</strong> Access key and secret key</li>
+        <li><strong>Native Library:</strong> Platform-specific JNI library (provided)</li>
+    </ul>
+
+    <h3>Pre-built Libraries Available</h3>
+    <p>We provide pre-built native libraries for:</p>
+    <ul>
+        <li>macOS (Intel &amp; Apple Silicon)</li>
+        <li>Ubuntu Linux (x64)</li>
+        <li>Windows 10+</li>
+        <li>Raspberry Pi</li>
+    </ul>
+    <p>Set the <code>java.library.path</code> JVM property to the <i>directory</i> containing the JNI library.</p>
+
+    <pre style="background-color: #f4f4f4; padding: 15px; border-left: 4px solid #2196f3; margin: 10px 0; font-family: 'Courier New', monospace;">// Pre-built libraries are located in:
+src/main/resources/lib/
+├── mac/
+├── ubuntu/
+├── windows/
+└── raspberry-pi/</pre>
+
+    <div style="background-color: #ffebee; padding: 15px; border-left: 4px solid #f44336; margin: 15px 0;">
+        <strong>Important:</strong> If your platform isn't listed, you'll need to build the JNI library yourself.
+        See the <a
+            href="https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-java/wiki/Building-and-Using-JNI">building
+        guide</a>.
+    </div>
+</div>
+
+<h3>Running the Demo</h3>
+<pre style="background-color: #f4f4f4; padding: 15px; border-left: 4px solid #2196f3; margin: 10px 0; font-family: 'Courier New', monospace;"># Build the project
+mvn clean compile assembly:single
+
+# Run the demo (replace with your values)
+java -classpath target/*jar-with-dependencies.jar \
+  -Daws.accessKeyId=YOUR_ACCESS_KEY \
+  -Daws.secretKey=YOUR_SECRET_KEY \
+  -Dkvs-stream=YOUR_STREAM_NAME \
+  -Djava.library.path=src/main/resources/lib/mac \
+  -Dlog4j.configurationFile=log4j2.xml \
+  com.amazonaws.kinesisvideo.demoapp.DemoAppMain</pre>
+
+<h2>Learn More</h2>
+<p>Click on any section below to dive deeper into the topics that interest you:</p>
+
+<details>
+    <summary>Result Handling &amp; State Transitions (How PIC Decides Next Steps)</summary>
```

Comment:
(nit) The drop down arrows are somewhat hard to notice, especially when one of them is already expanded. Can these be also wrapped in a heading format?

Reply:
I made the font bigger, changed the color to purple, and added a vertical space between as well